packages/server/database/updateStage.ts

```diff
@@ -8,7 +9,7 @@ import rMapIf from './rMapIf'
 const updateStage = async (
   meetingId: string,
   stageId: string,
-  phaseType: string,
+  phaseType: NewMeetingPhaseTypeEnum,
```

Comment:
Not related to this feature, but while working on it, I noticed this should be `NewMeetingPhaseTypeEnum` instead of `string`